gym/f110_gym/envs/f110_env.py

```diff
@@ -323,6 +376,34 @@ def update_map(self, map_path, map_ext):
             None
         """
         self.sim.set_map(map_path, map_ext)
+        self.set_map_params(map_path, map_ext)
```

Comment:
This seems redundant. Why couldn't you just grab everything from the Simulator() instance?

Reply:
I thought it would be easier to just reload the file, but it is probably more efficient to pull it from the simulator.agents[0].scan_simulator where the map is stored. 
I will adjust accordingly